task_processing/plugins/mesos/retrying_executor.py

```diff
@@ -102,9 +106,9 @@ def retry_loop(self):
     def run(self, task_config):
         if task_config.task_id not in self.task_retries:
             with self.task_retries_lock:
-                retry_value = self.retries - 1
+                retry_value = self.retries
                 if 'retries' in task_config:
```

Comment:
this construct is a bit confusing, let's go with simple if, same as on line 48: https://github.com/Yelp/task_processing/pull/81/files#diff-b78ffcc2ae6bff2d45f74a6fe10dc8ebR48

in fact just move it into separate function

    def task_or_executor_retries(self, task_config):
        return task_config.retries if 'retries' in task_config else self.retries